Quaver.Shared/Screens/Results/UI/Tabs/Overview/Graphs/ResultsOverviewGraphContainer.cs

```diff
@@ -272,14 +288,62 @@ private void CreateDevianceGraph()
+                var graphSize = new ScalableVector2(container.Width * 0.95f, container.Height * 0.95f);
+                switch (ConfigManager.ResultGraph.Value) // only generate the graphs if needed
                 {
-                    Parent = container,
-                    Alignment = Alignment.MidCenter,
-                    X = -5,
-                    Visible = true
-                };
+                    case ResultGraphs.Deviance:
+                        if (DevianceGraph == null)
+                        {
+                            DevianceGraph = new CachedHitDifferenceGraph(Map, Processor, graphSize)
+                            {
+                                Parent = container,
+                                Alignment = Alignment.MidCenter,
+                                X = -5,
+                                Visible = true
+                            };
+                        }
+                        // TODO how to make this better
+                        DevianceGraph.Visible = true;
+                        if (AccuracyGraph != null)
+                            AccuracyGraph.Visible = false;
+                        break;
+                    case ResultGraphs.Accuracy:
+                        if (AccuracyGraph == null)
+                        {
+                            AccuracyGraph = new CachedAccuracyGraph(Map, Processor, graphSize)
+                            {
+                                Parent = container,
+                                Alignment = Alignment.MidCenter,
+                                X = -5,
+                                Visible = true
+                            };
+                        }
+                        if (DevianceGraph != null)
+                            DevianceGraph.Visible = false;
+                        AccuracyGraph.Visible = true;
+
+                        var tooltipArea = new ImageButton(UserInterface.BlankBox)
+                        {
+                            Parent = container,
+                            Alignment = Alignment.MidRight,
+                            Size = container.Size,
+                            Alpha = 0f
+                        };
+
+                        const string tooltipText = "Course of accuracy throughout the map.\n" +
+                                                   "If the map was not completed, then it will additionally\n" +
+                                                   "show the accuracy if all subsequent hits had been\n" +
+                                                   "Marvelouses instead.";
+
+                        var game = GameBase.Game as QuaverGame;
+                        tooltipArea.Hovered += (sender, args) => game?.CurrentScreen?.ActivateTooltip(new Tooltip(tooltipText, ColorHelper.HexToColor("#5dc7f9")));
+                        tooltipArea.LeftHover += (sender, args) => game?.CurrentScreen?.DeactivateTooltip();
+
+                        break;
```

Comment:
I will restructure graph classes to be more generic and expandable as a list/array in my next PR once this is merged, to accomodate for the HP graph that's coming next. The current code only a temporary measure, I don't want to add more to this PR with more than it needs to.